content/docs/user-guide/assets/asset-processor/configuration.md

```diff
@@ -5,21 +5,55 @@ description: Configure Asset Processor for Open 3D Engine (O3DE) with AssetProce
-
-* Add new source asset types for Asset Processor.
-* Update existing file type rules.
-* Update the ignore list.
-* Add or remove target platforms.
-
-    {{< note >}}
-Asset Processor automatically builds assets for the host platform. For example, if Asset Processor is running on Windows, Asset Processor builds Windows assets even if pc is not enabled in the `.setreg` file.
-    {{< /note >}}
-
-* Add scan directories for Asset Processor.
-
-* Set the number of process jobs.
-* Specify *metafile fingerprinting*, that is, which files trigger related files to be rebuilt.
-
-The `Registry/AssetProcessorPlatformConfig.setreg` is well commented and has many syntax examples for the available configuration options. To add project-specific overrides, you can add an `AssetProcessorGamePlatformConfig.setreg` file to your projects’s `Registry` directory. This file is read after the engine `AssetProcessorGamePlatformConfig.setreg` configuration file and can have additional project-specific settings for the ignore list, platforms, and file types.
\ No newline at end of file
+You can use the exclusion system to block files matching patterns from asset processing. These exist under the registry path Amazon/AssetProcessor/Settings/ with a pattern that looks like:
+```json
+"Exclude _exclusion title_": {
+  "pattern": "regular expression pattern, anything that matches this pattern will be ignored by the Asset Processor"
+}
+```
+
+Each platform has a collection of tags applied to it, under the path `Amazon/AssetProcessor/Settings/`. These tags are then checked by other systems and don't do anything directly themselves. These follow the pattern:
+```json
+"Platform _platform name_": {
+  "tags:" "comma,separated,list,of,tags"
+}
+```
+
+Copy jobs are for source assets that should be copied directly to the cache as a product asset. Some assets don't need additional processing, so setting up a copy job this way is quicker than authoring an entire C++ based builder to make content of that type available to your game as an asset. Copy jobs are managed by a registry setting pattern under `Amazon/AssetProcessor/Settings/`, using the following pattern:
+```json
+"RC _copy job name_": {
+  // Matching type can be glob or pattern, do not use both. Pattern is for regular expression based matching, glob is useful for if you just want to match on a file extension.
+  "glob": "*.extension",
+  "pattern": "regex pattern to match this copy job", // It's either glob or pattern, not both
+  "productAssetType": "The UUID for the AZ::Data::AssetType for the product asset",
+  "params": "copy" or "skip", // Use "copy" for copy jobs, "skip" to skip processing. Skip is mostly used for platforms that don't need a particular asset type.
+  "platform tag": "copy" or "skip", // Copy or Skip for platforms matching this tag. See the earlier platform tag section for details.
+}
+```
```

Comment:
````suggestion
Copy jobs are for source assets that should be copied directly to the cache as a product asset. Some assets don't need additional processing, so setting up a copy job this way is quicker than authoring an entire C++ based builder to make content of that type available to your game as an asset. Copy jobs are managed by a registry setting pattern under `Amazon/AssetProcessor/Settings/`, using the following pattern:

```json
"RC <Copy job name>": {
  // Matching type can be glob or pattern, do not use both. Pattern is for regular expression based matching, glob is useful for if you just want to match on a file extension.
  "glob": "*.<extension>",
  "pattern": "<regex pattern to match this copy job>", // It's either glob or pattern, not both
  "productAssetType": "<The UUID for the AZ::Data::AssetType for the product asset>",
  "params": "<copy|skip>", // Use "copy" for copy jobs, "skip" to skip processing. Skip is mostly used for platforms that don't need a particular asset type.
  "platform tag": "<copy|skip>", // Copy or Skip for platforms matching this tag. See the earlier platform tag section for details.
}
```
````

Reply:
I've updated with this change plus some additional cleanup